srcpkgs/osm2pgsql/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'osm2pgsql'
 pkgname=osm2pgsql
 version=1.6.0
-revision=2
+revision=3
```

Comment:
The package has been revbumped for boost-devel in master, swallowing this revbump.

Reply:
updated; libgdal too.